homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -0,0 +1,177 @@
+"""
+Support for Waze travel time sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.waze_travel_time/
+"""
+import logging
+from datetime import timedelta
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['WazeRouteCalculator==0.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_DESTINATION = 'destination'
+CONF_ORIGIN = 'origin'
+CONF_OUTPUTS = 'outputs'
+CONF_REGION = 'region'
+CONF_UPDATE_INTERVAL = 'update_interval'
+
+TIME_UNIT = 'min'  # type: str
+DISTANCE_UNIT = 'km'  # type: str
+
+DEFAULT_NAME = 'Waze Travel Time'
+
+OUTPUT_TYPES = {
+    'duration':
+    ['Duration', TIME_UNIT, 'mdi:timer', 1],
```

Comment:
Style suggestion: Move this expression inline with `duration:` or at least indent it a bit - it's easier to parse in your head then.